### ISSUE TITLE
README

### ISSUE THREAD

**Issue by martin-henz** [source-academy/cadet-frontend#418]
Currently the README says:

**3. Copy the .env.example file as .env and set the variable REACT_APP_IVLE_KEY to contain your IVLE Lapi key.**

We need to explain what "variable" means here: shell environment variable, for example in BASH:
export REACT_APP_IVLE_KEY=&lt;here goes the key&gt;
(and then permanently edit: ~/.bashrc)

We also need to explain where NUS users get their LAPI key from: Go to:
https://ivle.nus.edu.sg/LAPI/default.aspx

So basically: go to https://ivle.nus.edu.sg/LAPI/default.aspx 
and copy the key into your ~/.bash_profile as follows:
export REACT_APP_IVLE_KEY=&lt;here goes the key&gt;

**Comment by jiachen247** [comment]
@martin-henz we actually are using [dotenv](https://www.npmjs.com/package/dotenv) to handle the exporting of env variables at runtime. All we have to do is update the .env file as already instructed in the readme.

I do think its a good idea to let students know where they can get their hands on a valid IVLE api key.
